src/GeneralStateTestsFiller/stCreateTest/CreateResultsFiller.yml

```diff
@@ -0,0 +1,171 @@
+CreateResults:
+
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x20000'
+    currentGasLimit: 0x100000000
+    currentNumber: "1"
+    currentTimestamp: "1000"
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  _info:
+    comment: Ori Pomerantz qbzzt1@gmail.com
+
+
+  pre:
+
+    cccccccccccccccccccccccccccccccccccccccc:
+      balance: '0x0ba1a9ce0ba1a9ce'
+      code:   | 
+        {
+
+          ; Variables are 0x20 bytes (= 256 bits) apart, except for
+          ; code buffers that get 0x100 (256 bytes)
+          (def 'creation          0x100)
+          (def 'callType          0x120)
+          (def 'contractCode      0x200)
+          (def 'constructorCode   0x300)
+          (def 'extCode           0x400)
+          (def 'contractLength    0x520)
+          (def 'constructorLength 0x540)
+          (def 'extLength         0x560)
+          (def 'addr1             0x600)
+          (def 'addr2             0x620)
+          (def 'callRet           0x640)
+
+          ; Other constants
+          (def 'NOP 0)   ; No OPeration
+
+          ; Understand the input.
+          [creation]       (shr $ 0 248)
+          [callType]       (and (shr $ 0 240) 0xFF)
+
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+          ; Create the contract and a constructor to pass to CREATE[2]
+          ;
+
+
+          [constructorLength] (lll
+              (return contractCode
+                     ; write the contract to constructor memory 
+                     (lll 
+                         (mstore 0xFF 0xFF)
+                         contractCode
+                     )
+              )
+              constructorCode
+          )
+
+          ; Write the contract to our own memory (as the 0xcc...cc contract)
+          [contractLength] (lll 
+             (mstore 0xFF 0xFF)
+             contractCode
+           )
+
+
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+          ; Actually create the contract
+          ;
+          (if (= @creation 1)
+            [addr1] (create 0 constructorCode @constructorLength)
+            [addr1] (create2 0 constructorCode @constructorLength 0x5a17)
+          )
+
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+          ; RETURNDATASIZE should be zero
+          ;
+
+          [[0x10]] (returndatasize)
+
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+          ; EXTCODE should match what we returned in the constructor
+          ;
+
+          [extLength] (extcodesize @addr1)
+          (extcodecopy @addr1 extCode 0 @extLength)
+
+          [[0x11]] (- @contractLength @extLength)
+          [[0x12]] (- @contractCode   @extCode)
+
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+          ; CALL the created contract
+
+          (if (= @callType 1)
+             [callRet] (call 0xFFFF @addr1 0 0 0 0 0)
```

Comment:
put some code in contract like 
sstore [100] = 12 
so that you know that created contract got executed
(I do not suggest to touch the memory in created contract code and return because it will effect the retrundatasize/returndatacopy)

Reply:
The problem with just modifying storage is that I don't necessarily know the contract address (it's consistent right now, even with CREATE, but I don't think that's promised). So instead I'm going to call a contract with a fixed address from the created contract.